src/odb/src/zutil/util.cpp

```diff
@@ -221,10 +240,19 @@ void cutRows(dbBlock* block,
     }
   }
 
-  const int final_sites_count
-      = std::accumulate(rows.begin(), rows.end(), 0, [](int sum, dbRow* row) {
-          return sum + row->getSiteCount();
-        });
+  const std::int64_t final_sites_count = std::accumulate(
+      rows.begin(),
+      rows.end(),
+      (std::int64_t) 0,
+      [&](std::int64_t sum, dbRow* row) {
+        if (hasOverflow(sum, (std::int64_t) row->getSiteCount())) {

```

Comment:
We should not need the overflow check when using an int64_t

Reply:
I've removed the check, but kept the `hasOverflow` function in case we need it in the future.